templates/desktop/index.html

```diff
@@ -16,6 +17,82 @@
 
 {% block content %}
 
+  {%- call(slot) tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=true) -%}
+    {%- if slot == 'title' -%}
+      <h2>H2 - up to two lines; ideally one.</h2>
```

Comment:
If we're loading the macros from node modules, why do we need to duplicate them here? Is this just to test/POC this?